package.json

```diff
@@ -6,7 +6,6 @@
     "node": "16 || 18"
   },
   "scripts": {
-    "preinstall": "npx only-allow yarn",
```

Comment:
is npx installed and available downstream too? need to make sure it's in the list of requirements that cachito (or an RPM) installs, or this will fail.

Reply:
`npx` should be always installed with `npm` since `npm` v5.2.

Why was this removed?

Reply:
Hi so when we ran `yarn install` with the `--ignore-scripts` flag, the required `isolated-vm` module doesn't get built. But when we disable the `--ignore-scripts` flag, the following error occurs:
```bash
RUN $YARN install --frozen-lockfile --network-timeout 600000
yarn install v1.22.19
$ npx only-allow yarn
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
gelog: <https://github.com/npm/cli/releases/tag/v9.8.0>
npm notice Run `npm install -g npm@9.8.0` to update!
npm notice 
npm ERR! code ENOENT
npm ERR! syscall lstat
npm ERR! path /opt/app-root/src/.npm-global
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, lstat '/opt/app-root/src/.npm-global'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /opt/app-root/src/.npm/_logs/2023-07-12T14_24_52_707Z-debug-0.log
error Command failed with exit code 254.
Error: building at STEP "RUN $YARN install --frozen-lockfile --network-timeout 600000": while running runtime: exit status 254
```

Reply:
@tumido do you have any ideas how to resolve this without removing the `preinstall` script?